test/tests/functional/pbs_qmgr.py

```diff
@@ -98,11 +100,19 @@ def test_listcmd_whitespaces(self):
             long_string += blah
             # Create a new vnode
             attrs = {ATTR_rescavail: 2}
```

Comment:
I missed to add a comment here in last round. ATTR_rescavail is just "resoures_available" you do need ".<resource_name>" appended to it to make it meaningful. I am a little surprised to know that pbs_mom is conveniently ignoring such abnormality in vnodedef file.